### ISSUE TITLE
MAC comparison failed. Key or payload has changed.

### ISSUE THREAD

**Issue by mihalyr** [bitwarden/clients#11074]
### Steps To Reproduce

Run for example `bw list items`

### Expected Result

A list of items

### Actual Result

```
➜  ~ bw list items
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
MAC comparison failed. Key or payload has changed.
[{"passwordHistory":null,"revisionDate":
```

### Additional Context

Why does bw CLI return a bunch of MAC failed lines when I try to list my vault items? Does this mean it fails to retrieve some of the items? Which ones? 

This is reopening of https://github.com/bitwarden/clients/issues/11072 because customer service for this product closed it in a hurry without asking questions. Not sure if this is what I expect from a paid service. 

CLI just downloaded fresh from your website. 

### Operating System

Linux

### Operating System Version

Fedora 40

### Shell

Zsh

### Build Version

2024.8.2

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by sammbw** [comment]
Hi there!

Thank you for your report, it seems like it is a duplicate of this one: https://github.com/bitwarden/clients/issues/11072

You will need to contact the support team directly using the form below to troubleshoot this issue, as this is not a client related bug.

Contact Bitwarden: https://bitwarden.com/contact/

This issue will now be closed.

Thanks!

**Comment by mihalyr** [comment]
@sammbw I think your website has some issues, before I opened this ticket I went to https://bitwarden.com/help/ and tried to click on "Contact support":

![image](https://github.com/user-attachments/assets/9038c5e5-e467-4f96-b66f-2a7aa436eb9d)

But after clicking on it, all it did was to disappear and do nothing:

![image](https://github.com/user-attachments/assets/e23a86a2-2d39-4568-88b8-1f7ad2415709)

So next I used the "File a bug report" menu from the desktop app that led me to this GitHub issue that you keep closing. This is quite a disappointing experience, you might want to consider some improvements. 

The link you shared (https://bitwarden.com/contact/) does give me a contact form, so I went back to you website to see how can I navigate to the page and realized it is hidden in the bottom of the footer. There is certainly room for improvement here.
